_posts/2024-08-27-updating-member-details.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  A better way to update union member details
+description:
+  Making it easy for members to update their details can result in better retention, more revenue and happier members. Here's how Join Together can help you do this.
+og_image_path: 
+date:   2024-08-27 11:19:13 +0000
+---
+
+We’ve worked with unions for years, and heard all sorts of horror stories about the difficulties of staying in touch with members. To try and fix this, we’ve seen unions taking a variety of approaches. These include:
+
+- Asking for updated details by post or phone
+- Collecting information via forms or spreadsheets, usually just before a ballot takes place
+- Getting members to log in to a membership portal on the union website
+
+Each of these has different challenges and costs. Calls and letters are expensive to make or send, get low response rates and often interrupt what members are doing at a particular moment (”I’ll do that later” and “who’s that number calling me?”). 
+
+Collecting information via spreadsheets leads to enormous data protection risks, as well as problems with data entry, as files are passed from person to person, computer to computer, usually ending with a messy import or copy/paste/fix the mistakes process, slowing the whole thing down.
+
+Finally, union membership portals are challenging for their users. The typical experience involves using data members probably don’t know (their membership number or a password they never use, leading to a password reset or an email to the membership team) before logging into a piece of software they’re unfamiliar with to perform a task they can’t be sure they’ll be able to complete.
```

Comment:
This is sweet. They read well in this order (calls/letters->spreadsheets->portals). Is this the order that's most common -> least common?